INI settings for 'Hasbro Family Game Night 2'

jack-1313 opened the pull request:
Fixes :
https://bugs.dolphin-emu.org/issues/12534

Rumi-Larry commented:
Cool, but the pr title should have the name of the game :)

mbc07 commented:
If Sync GPU Thread can fix this, I think it should be preferred over disabling Dual Core. As bad as Dual Core is, it can still provide nice speed boosts especially for people with low end machines...

jack-1313 commented:
> Cool, but the pr title should have the name of the game :)

I hope it is fine now.
 (.❛ ᴗ ❛.)

JMC47 commented:
I saw conflicting reports that syncGPU may not always fix it when on the unofficial discord.

jack-1313 commented:
> If Sync GPU Thread can fix this, I think it should be preferred over disabling Dual Core. As bad as Dual Core is, it can still provide nice speed boosts especially for people with low end machines...

So, I tested using “Sync GPU Thread” enabled, and this fixed this issue. _Although, it is not much faster than single core as I wasn’t able to notice any significance difference._

But still I have made the required changes in the file.
Hopefully, everything is ok now.

jack-1313 commented:
> I saw conflicting reports that syncGPU may not always fix it when on the unofficial discord.

For me both (Disable Dual Core and Sync GPU Thread) worked, it’s now up to you guys to consider which option would be the best. Maybe someone else should test it as well. 

@JMC47 
When did you get that feedback? If it was over one month ago, then it could be me, as I had talked to you before filing the bug.

Rumi-Larry commented:
> 
> 
> > Cool, but the pr title should have the name of the game :)
> 
> I hope it is fine now.
> (.❛ ᴗ ❛.)

I suggest "INI settings for 'Hasbro Family Game Night 2'"

AdmiralCurtiss commented on Data/Sys/GameSettings/R6X.ini:
```diff
@@ -0,0 +1,9 @@
+ SyncGPU = True
+
+[Video_Settings]
+ # The intro logo shutters. 
```
I assume you mean 'stutters', not 'shutters'.

jack-1313 commented:
I have corrected the spelling mistake, and **for now, I have reverted back to "Disable Dual Core"**, because of this:-

> I saw conflicting reports that syncGPU may not always fix it when on the unofficial discord.

jack-1313 commented:
Can this PR be merged? 

“Disable Dual Core” and “Safe Texture Cache Accuracy” completely solve the issue.

As for “Sync GPU”, we don’t know if it is able to fix the hang for all devices. Until we get more details, this GameINI seems good to me. And if we later get the confirmation that “Sync GPU” fixes, then I will make a new PR for the required changes.

_Also, I tried my best to merge all my 4 commits, but since I am new to GitHub, I wasn’t able to do so. Hopefully you guys can use the “Squash and Merge” option._

Rumi-Larry commented:
The title is good, but you missed the second single quote

jack-1313 commented:
> The title is good, but you missed the second single quote

Oh, sorry for that. My first PR has truly turned to be a disaster with so many mistakes. 

Hope everything is now good to go.

mbc07 commented:
A simple change like this is very unlikely to get merged until you squash your commits...

jack-1313 reviewed (commented):
I think this is the squashed one, the only which needs to be merged?

I am confused, have I done anything wrong?

JosJuice commented:
It looks like you first created a squashed commit, and then merged/pulled origin. You need to do it without merging/pulling origin. If the reason why you merged/pulled origin is because git push wouldn't let you push otherwise, you need to use the `-f` option to `git push` to override that.

jack-1313 commented:
When I try squashing all the 6 commits, I get this error-
"Unable to squash. Squashing replays all commits up to the last one required for the squash. A merge commit cannot exist among those commits."

Do I need commit all 6, or only the 3 that I had earlier tried to squash?

JosJuice commented:
Try `git reset --hard 8f107191cbed3d993a2db41440e24c77449491cd`

This will bring you back to how things were right after making the 8f107191cbed3d993a2db41440e24c77449491cd commit, before making the merge commit.

jack-1313 commented:
@JosJuice 

When I try to `git push origin`, it asks for `git pull`, so I do that also.

After that it shows this message
"Please enter a commit message to explain why this merge is necessary, especially if it merges an updated upstream into a topic branch.
Lines starting with '#' will be ignored, and an empty message aborts the commit."

I am stuck here, can you please help what to do now?

JosJuice commented:
Like I mentioned before, you shouldn't use `git pull`. Instead, use `git push -f` in place of `git push`.

jack-1313 commented:
So, finally, somehow I am able to squash all my commits into one with JosJuice's help.
Let me know if there is something else that also needs to be done.

JMC47 commented:
@dolphin-emu-bot rebuild

JMC47 commented:
I double checked single core vs dualcore + syncGPU performance in a few games just in case, and it doesn't seem to be a major difference.  If anything, the safe texture cache is a harder hit, but it appears to be necessary to put it to safe.  Seems fine to me and the game isn't really playable in master anyway unless you use single core.
